facet-jsonschema/Cargo.toml

```diff
@@ -20,9 +20,7 @@ keywords = [
 categories = ["development-tools", "encoding"]
 
 [dependencies]
-facet-core = { version = "0.16.0", path = "../facet-core" }
-facet-derive = { version = "0.16.0", path = "../facet-derive" }
-facet-reflect = { version = "0.16.0", path = "../facet-reflect" }
```

Comment:
> @epage warned specifically against this so I'm going to go with no :(

The difference here is that `facet-jsonschema` depends on the derive while the other cases do not.  A worthwhile question is how important using the derive is to this crate but I would consider jsonschema lower priority than cases like json.

I do wonder what made the difference in build times.  This should have made the build a little more serial but have no other impact.  I suspect it has to do with `cargo hack` and either it added more cases for some reason or it built things in an order that had better reuse of cache with feature activations (unsure how much of that gets thrown out for local builds or reused).